tests/cases/debugger_hlsl_line_stepping_no_inline_param.amber

```diff
@@ -114,57 +112,45 @@ VS_OUTPUT main(float4 pos : POSITION,
          %53 = OpExtInst %void %1 DebugLocalVariable %14 %43 %41 8 13 %52 FlagIsLocal
          %44 = OpExtInst %void %1 DebugTypeMember %12 %47 %41 2 3 %43 %uint_0 %uint_128 FlagIsProtected|FlagIsPrivate
          %45 = OpExtInst %void %1 DebugTypeMember %13 %47 %41 3 3 %43 %uint_128 %uint_128 FlagIsProtected|FlagIsPrivate
-               OpLine %7 6 1
        %main = OpFunction %void None %36
          %54 = OpLabel
          %79 = OpExtInst %void %1 DebugScope %52
-               OpLine %7 8 13
          %71 = OpVariable %_ptr_Function_VS_OUTPUT Function
          %65 = OpExtInst %void %1 DebugDeclare %53 %71 %40
          %80 = OpExtInst %void %1 DebugScope %49
          %72 = OpVariable %_ptr_Function_VS_OUTPUT Function
          %81 = OpExtInst %void %1 DebugScope %42
-               OpLine %7 6 23
 %param_var_pos = OpVariable %_ptr_Function_v4float Function
          %62 = OpExtInst %void %1 DebugDeclare %50 %param_var_pos %40
-               OpLine %7 7 23
 %param_var_color = OpVariable %_ptr_Function_v4float Function
          %63 = OpExtInst %void %1 DebugDeclare %51 %param_var_color %40
-               OpLine %7 6 23
          %56 = OpLoad %v4float %in_var_POSITION
                OpStore %param_var_pos %56
-               OpLine %7 7 23
          %57 = OpLoad %v4float %in_var_COLOR
                OpStore %param_var_color %57
          %82 = OpExtInst %void %1 DebugScope %52
-               OpLine %7 9 14
          %73 = OpLoad %v4float %param_var_pos
+               OpLine %7 8 13
```

Comment:
`VS_OUTPUT vout;` doesn't actually do anything (hence nothing between this `OpLine` and the next), but needs to come after the parameters are primed with values.